applications/nrf5340_audio/src/audio/CMakeLists.txt

```
@@ -8,6 +8,7 @@ target_sources(app PRIVATE
 	       ${CMAKE_CURRENT_SOURCE_DIR}/audio_system.c
 	       ${CMAKE_CURRENT_SOURCE_DIR}/audio_datapath.c
 	       ${CMAKE_CURRENT_SOURCE_DIR}/sw_codec_select.c
+	       ${CMAKE_CURRENT_SOURCE_DIR}/le_audio_rx.c
```

```suggestion
	       ${CMAKE_CURRENT_SOURCE_DIR}/audio_rx.c
```
Maybe this is better, I feel "le_audio_XXX" is more suited for the Bluetooth modules

---

Discuss